src/lib/parser_impl.c

```diff
@@ -557,6 +610,10 @@ uint8_t _getNumItems(parser_context_t *c, parser_tx_t *v) {
         case registryUnfreezeNode:
             itemCount += 1;
             break;
+        case registryRegisterEntity:
+            // REVIEW: only 2 because we ignore the raw_untrusted_value
```

Comment:
we should actually deserialize the raw_untrusted_value with the code we already have for entities and show the entity